### PR TITLE
[ES] Add `osu!catch` and `osu!taiko`

### PR DESCRIPTION
**Pull request by virasakd**
The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
 (translations only) The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)
 
I have translated these two documents, "osu!catch" and "osu!taiko," into Spanish.

### CODE REVIEW

**Review by githacki — CHANGES_REQUESTED** [review]
Fix remark

File: wiki/Game_mode/osu!catch/es.md [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+*Referencia a [Página de Skinning de osu!¡Todo el mundo!](/wiki/Skinning/osu!Catch) para toda la información.*
+
+
+## Beatmapping
+
+### ¡Oso! Notas de conversión
+
+- La ubicación horizontal del eje determina el punto de caída de las frutas (circuito de golpes) o las pistas de frutas (slider).
+- La apariencia se basa en el tiempo establecido para limpiarlo, como es habitual.
+- La velocidad de la pista de frutas es similar a la velocidad de las bolas de deslizamiento.
```
Comment (agatemosu) [inline wiki/Game_mode/osu!catch/es.md:104]:
```suggestion
- La velocidad de la corriente de jugo es similar a la velocidad de las bolas del slider.
```

File: wiki/Game_mode/osu!catch/es.md [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+- Spinner crea una lluvia de plátano.
```
Comment (agatemosu) [inline wiki/Game_mode/osu!catch/es.md:105]:
```suggestion
- El spinner crea una lluvia de plátanos.
```

File: wiki/Game_mode/osu!catch/es.md [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+
+**Notas:** Los hiperfructos se generan automáticamente.
```
Comment (agatemosu) [inline wiki/Game_mode/osu!catch/es.md:107]:
```suggestion
**Nota:** Las hiperfrutas se generan automáticamente.
```

File: wiki/Game_mode/osu!catch/es.md [hunk 4]
```diff
@@ -0,0 +1,126 @@
...
+
+## Trivia
```
Comment (agatemosu) [inline wiki/Game_mode/osu!catch/es.md:109]:
```suggestion
## Curiosidades
```

File: wiki/Game_mode/osu!catch/es.md [hunk 5]
```diff
@@ -0,0 +1,126 @@
...
+
+### El Gameplay
+
+- [Auto]( / wiki / Gameplay / Game_modifier / Auto) siempre recogerá todas las bananas.
```
Comment (agatemosu) [inline wiki/Game_mode/osu!catch/es.md:113]:
```suggestion
- [Auto](/wiki/Gameplay/Game_modifier/Auto) siempre recogerá todos los plátanos.
```

File: wiki/Game_mode/osu!catch/es.md [hunk 6]
```diff
@@ -0,0 +1,126 @@
...
+- Si `Hit-error` está habilitado, siempre utilizará `Color`.
```
Comment (agatemosu) [inline wiki/Game_mode/osu!catch/es.md:114]:
```suggestion
- Si `Hit-error` está habilitado, siempre utilizará `Colour`.
```

File: wiki/Game_mode/osu!catch/es.md [hunk 7]
```diff
@@ -0,0 +1,126 @@
...
+- El atrapador puede moverse alrededor durante cualquier pausa.
```
Comment (agatemosu) [inline wiki/Game_mode/osu!catch/es.md:115]:
```suggestion
- El catcher puede moverse alrededor durante cualquier pausa.
```

File: wiki/Game_mode/osu!catch/es.md [hunk 8]
```diff
@@ -0,0 +1,126 @@
...
+- El catcher tiene una animación de Miss, mucho como osu!Animación de Miss Taiko Pippidon.
```
Comment (agatemosu) [inline wiki/Game_mode/osu!catch/es.md:116]:
```suggestion
- El catcher tiene una animación de fallo, muy parecida a la animación de fallo de pippidon en osu!taiko.
```

File: wiki/Game_mode/osu!catch/es.md [hunk 9]
```diff
@@ -0,0 +1,126 @@
...
+- El primer catcher por defecto fue una versión chibificada de Ryuuta Ippongi (de la serie [ouendan] (https://en.wikipedia.org/wiki/Osu!¡Tataque!_Ouendan)) hasta 2014, que fue reemplazado por el catcher y mascota actual, [Yuzu](/wiki/Mascots#yuzu)
```
Comment (agatemosu) [inline wiki/Game_mode/osu!catch/es.md:117]:
```suggestion
- El primer catcher por defecto fue una versión chibificada de Ryuuta Ippongi (de la [serie ouendan](https://es.wikipedia.org/wiki/Osu!_Tatakae!_Ouendan)) hasta 2014, que fue reemplazado por el catcher y mascota actual, [Yuzu](/wiki/Mascots#yuzu)
```

File: wiki/Game_mode/osu!catch/es.md [hunk 10]
```diff
@@ -0,0 +1,126 @@
...
+- Cuando se juega por [Auto] (/wiki/Gameplay / Game_modifier/Auto), el nombre del jugador será *salad!*.
```
Comment (agatemosu) [inline wiki/Game_mode/osu!catch/es.md:118]:
```suggestion
- Cuando se juega con [Auto](/wiki/Gameplay/Game_modifier/Auto), el nombre del jugador será *salad!*.
```

File: wiki/Game_mode/osu!catch/es.md [hunk 11]
```diff
@@ -0,0 +1,126 @@
...
+
+### Historia
+
+![osu!catch logo en los modos especiales](/wiki/shared/Ctb_logo.jpg "Catch the Beat en modo especial")
```
Comment (agatemosu) [inline wiki/Game_mode/osu!catch/es.md:122]:
```suggestion
![Logo de osu!catch en modo especial](/wiki/shared/Ctb_logo.jpg "Logo de Catch the Beat en modo especial")
```

File: wiki/Game_mode/osu!catch/es.md [hunk 12]
```diff
@@ -0,0 +1,126 @@
...
+
+- Hasta el 10 de abril de 2012 (2012-04-10), los mapas de osu!catch no podían ser clasificados ni incluidos en mapas clasificados. Sin embargo, esto ha cambiado ahora, y los creadores de mapas pueden hacer dificultades específicas de osu!catch, siempre que cumplan con las directrices. Estas directrices son en gran medida un trabajo en curso, pero a medida que los mapas de osu!catch sean clasificados y reciban comentarios de los jugadores, las directrices evolucionarán para ser tan exhaustivas y completas como las directrices generales de osu!.
```
Comment (agatemosu) [inline wiki/Game_mode/osu!catch/es.md:124]:
```suggestion
- Hasta el 10 de abril de 2012 (10/4/2012), los mapas de osu!catch no podían ser clasificados ni incluidos en mapas clasificados. Sin embargo, esto ha cambiado ahora, y los creadores de mapas pueden hacer dificultades específicas de osu!catch, siempre que cumplan con las directrices. Estas directrices son en gran medida un trabajo en curso, pero a medida que los mapas de osu!catch sean clasificados y reciban comentarios de los jugadores, las directrices evolucionarán para ser tan exhaustivas y completas como las directrices generales de osu!.
```

File: wiki/Game_mode/osu!catch/es.md [hunk 13]
```diff
@@ -0,0 +1,126 @@
...
+- El primer beatmap clasificado que contiene al menos una dificultad de osu!catch es [Yousei Teikoku - Dare so Ka no Gekka (TV Size) (NatsumeRin), con el osu!la dificultad para capturar a los huéspedes mapeado por Urano](https://osu.ppy.sh/beatmapsets/13676).
+- ¡Primer clasificado OSU!catch-only beatmap es [Río de Janeiro - Córdoba](https://osu.ppy.sh/beatmapsets/91485).
```
Comment (agatemosu) [inline wiki/Game_mode/osu!catch/es.md:126]:
```suggestion
- El primer beatmap clasificado solo de osu!catch es [Rita - Hajimari no Toki (Deif)](https://osu.ppy.sh/beatmapsets/91485).
```

**Review by agatemosu — COMMENTED** [review]
Part 1

File: wiki/Game_mode/osu!taiko/es.md [hunk 14]
```diff
@@ -0,0 +1,145 @@
...
+  - 4 veces la longitud del slider, o...
+  - 8 veces la longitud del slider en canciones que son iguales o inferiores a 125 BPM.
+- A diferencia de los otros modos de juego, *Kiai Time* tiene un efecto en las puntuaciones porque se refiere al *"Go-Go Time"* en *Taiko no Tatsujin*. Mientras *Kiai Time* está activo, el tambor en la parte superior izquierda cambia de animación (llamado *pippidon* o *Don*/*Katsu* en *Taiko no Tatsujin*), el campo de juego tiene un gradiente de fondo y la zona de golpe gana un gráfico de fuego alrededor de él.
+  - Además, todas las notas de éxito obtienen un multiplicador de puntuación de 1.2x, incluidas notas amarillas largas, excepto para los hits en un shaker. (the final hit is still multiplied).
+- El personaje mascota de osu!taiko es[pippidon](/wiki/Mascots#pippi).
+- Cuando se juega por [Auto](/wiki/Gameplay/Game_modifier/Auto), el nombre del jugador será *mekkadosu!*.
+
+### Historia
+
+![El logo de osu!taiko en los modos especiales.](img/Taiko_logo.jpg "Logo de Taiko en el Modo Especial.")
```
Comment (agatemosu) [inline wiki/Game_mode/osu!taiko/es.md:140]:
```suggestion
![Logo de osu!taiko en modo especial](img/Taiko_logo.jpg "Logo de Taiko en modo especial")
```

File: wiki/Game_mode/osu!taiko/es.md [hunk 15]
```diff
@@ -0,0 +1,145 @@
...
+
+
+- La opción `Usar skin Taiko para modo Taiko` en la barra lateral de Opciones en la sección de skins utilizará los elementos de la carpeta `taiko` cuando juegue en *osu!taiko*, independientemente de los elementos taiko de la skin actual. La carpeta usada para mantener la piel *[Taiko por LuiginHann](https://osu.ppy.sh/community/forums/topics/41319)*, que se puede descargar desde el ahora desaprobado `osume.exe` (un archivo de actualización de osu! cuando sea necesario actualizar) en la pestaña `Skin`.
+- El primer beatmap clasificado que contiene al menos un osu!la dificultad de taiko es [Taiko no Tatsujin - Saitama2000 (Kharl)](https://osu.ppy.sh/beatmapsets/210)
```
Comment (agatemosu) [inline wiki/Game_mode/osu!taiko/es.md:144]:
```suggestion
- El primer beatmap clasificado que contiene al menos una dificultad de osu!taiko es [Taiko no Tatsujin - Saitama2000 (Kharl)](https://osu.ppy.sh/beatmapsets/210)
```

File: wiki/Game_mode/osu!taiko/es.md [hunk 16]
```diff
@@ -0,0 +1,145 @@
...
+- ¡Primer clasificado OSU!beatmap es [Mutsuhiko Izumi - Red Goose (lepidopodus)](https://osu.ppy.sh/beatmapsets/55920)
```
Comment (agatemosu) [inline wiki/Game_mode/osu!taiko/es.md:145]:
```suggestion
- El primer beatmap clasificado de osu!taiko es [Mutsuhiko Izumi - Red Goose (lepidopodus)](https://osu.ppy.sh/beatmapsets/55920)
```

File: wiki/Game_mode/osu!taiko/es.md [hunk 17]
```diff
@@ -0,0 +1,145 @@
...
+  - Además, todas las notas de éxito obtienen un multiplicador de puntuación de 1.2x, incluidas notas amarillas largas, excepto para los hits en un shaker. (the final hit is still multiplied).
+- El personaje mascota de osu!taiko es[pippidon](/wiki/Mascots#pippi).
+- Cuando se juega por [Auto](/wiki/Gameplay/Game_modifier/Auto), el nombre del jugador será *mekkadosu!*.
+
+### Historia
+
+![El logo de osu!taiko en los modos especiales.](img/Taiko_logo.jpg "Logo de Taiko en el Modo Especial.")
+
+
+- La opción `Usar skin Taiko para modo Taiko` en la barra lateral de Opciones en la sección de skins utilizará los elementos de la carpeta `taiko` cuando juegue en *osu!taiko*, independientemente de los elementos taiko de la skin actual. La carpeta usada para mantener la piel *[Taiko por LuiginHann](https://osu.ppy.sh/community/forums/topics/41319)*, que se puede descargar desde el ahora desaprobado `osume.exe` (un archivo de actualización de osu! cuando sea necesario actualizar) en la pestaña `Skin`.
```
Comment (agatemosu) [inline wiki/Game_mode/osu!taiko/es.md:143]:
```suggestion
- La opción `Utilizar la skin de Taiko para Taiko` en la barra lateral de opciones en la sección de skins utilizará los elementos de la carpeta `taiko` cuando juegue en *osu!taiko*, independientemente de los elementos taiko de la skin actual. La carpeta usada para mantener la skin *[Taiko por LuiginHann](https://osu.ppy.sh/community/forums/topics/41319)*, que se puede descargar desde el ahora desaprobado `osume.exe` (un archivo para actualizar osu! para cuando sea necesario actualizar) en la pestaña `Skin`.
```

**Review by agatemosu — COMMENTED** [review]
Part 2

**Comment by cl8n** [comment]
see above

**Pull request closed without merging**